src/js/index.js

```diff
@@ -286,7 +300,7 @@ export const hpe = deepFreeze({
     },
     error: {
       background: {
-        color: { light: '#FF404033', dark: '#FF40404D' },
+        color: { light: '#FC6161', dark: '#C54E4B' },
```

Comment:
These look suspect. They are the colors of 'red' and 'red!', which are correct-ish. The visual color has these colors plus a 24% and 36% opacity. Ideally we would reference by name rather than use HEX here. Potentially could have a function which receives (colorname, opacity) and returns a hexvalue with opacity.

```suggestion
        color: { light: '#FC61613D', dark: '#C54E4B5C' },
```

Reply:
This may be related to your commentary in the PR description, but wasn't exactly sure.